source/module_esolver/esolver_ks_lcao.cpp

```diff
@@ -1108,12 +1107,11 @@ void ESolver_KS_LCAO<TK, TR>::after_scf(const int istep)
     }
 
     // 4) write density matrix
-    if (this->LOC.out_dm)
+    if (INPUT.out_dm)
     {
-        for (int is = 0; is < GlobalV::NSPIN; is++)
-        {
-            this->create_Output_DM(is, istep).write();
-        }
+        ModuleBase::WARNING_QUIT("ESolver_KS_LCAO",
```

Comment:
It seems that this is a temporary solution, can we use Hcontainer to output density matrix in dense format?

Reply:
No, we cannot get the dense matrix from HContainer now.

Reply:
Class DensityMatrix can return the DMK in dense format. But, here is just remove the function of out_dm, we do not need to use HContainer to output the DMK.